swap/swap_test.go

```diff
@@ -1508,3 +1564,142 @@ func TestAvailableBalance(t *testing.T) {
 	}
 
 }
+func TestBouncedCheque(t *testing.T) {
+	testBackend := newTestBackend(t)
+	defer testBackend.Close()
+	// create both test swap accounts
+	creditorSwap, clean1 := newTestSwap(t, beneficiaryKey, testBackend)
+	debitorSwap, clean2 := newTestSwap(t, ownerKey, testBackend)
+	defer clean1()
+	defer clean2()
+
+	testAmount := DefaultPaymentThreshold + 42
+
+	ctx := context.Background()
+	err := testDeploy(ctx, creditorSwap, int256.Uint256From(0))
```

Comment:
since you are only obtaining `err` here, you can do something like:

```golang
if rr := testDeploy(ctx, creditorSwap, int256.Uint256From(0)); err != nil{
    t.Fatal(err)
}
```

apply this also to the rest of the `TestBouncedCheque` function when possible